src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -103,13 +103,14 @@ static void usage()
             << "  --device <device path>  Specify nbd device path (/dev/nbd{num})\n"
             << "  --exclusive             Forbid writes by other clients\n"
             << "  --max_part <limit>      Override for module param max_part\n"
-            << "  --nbds_max <limit>      Override for module param nbds_max\n"
+            << "  --nbds_max <limit>      Override for module param nbds_max, will be ignored if --legacy is absent\n"
             << "  --quiesce               Use quiesce callbacks\n"
             << "  --quiesce-hook <path>   Specify quiesce hook path\n"
             << "                          (default: " << Config().quiesce_hook << ")\n"
             << "  --read-only             Map read-only\n"
             << "  --timeout <seconds>     Set nbd request timeout\n"
-            << "  --try-netlink           Use the nbd netlink interface\n"
+            << "  --legacy                Use the nbd ioctl interface for old kernels\n"
```

Comment:
```suggestion
            << "  --legacy                Use the nbd ioctl interface (old kernels)\n"
```

Reply:
It seems we do not use articles in other options description. May be we can skip it here to just to make the text more compact?

Reply:
Since the `--nbds-max` already in the `doc/man/8/rbd-nbd.rst`, and we have changed it here, so we need to update it.